cookbook/attributes/default.rb

```diff
@@ -14,21 +14,8 @@
   },
 }
 
-default[:mesos] = {
-  :package_version => "0.21.0-1.0.ubuntu1404",
-  :common => {
-    :ip => private_ip,
-  },
-  :master => {
-    :cluster => 'vagrant-singularity',
-  },
-  :slave => {
-    :containerizers => 'docker,mesos',
-    :switch_user => 'false'
```

Comment:
How are these options actually being used? They don't seem to appear anywhere in the cookbook...